pyfixest/report/visualize.py

```diff
@@ -223,7 +245,8 @@ def coefplot(
         idxs = df.index
     df = df.loc[idxs, :].reset_index()
 
-    return _coefplot(
+    _coefplot(
```

Comment:
The `return` statement is missing, same as above.

Reply:
fixed